Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -29,8 +29,8 @@
   # Defines for default states.  These can be changed on the command line.
   # -D FLAG=VALUE
   #
-!ifndef DEBUGGER_ENABLED
-  DEFINE DEBUGGER_ENABLED               = FALSE
+!ifndef FORCE_DXE_DEBUGGER

```

Comment:
What is the benefit of this? It seems if we have the configuration pei module there is no need to force enable.

Reply:
Yeah, agreed. I will set this to control the initial breakpoint timeout only.

Reply:
I think what you want is to leave the timeout alone and have a flag to control whether you get the initial breakpoint, by seeing bit 0 of the flags control:  https://github.com/cfernald/mu_feature_debugger/blob/62f972757843a1c9b583efd681ddac6f65ac11bf/DebuggerFeaturePkg/DebuggerFeaturePkg.dec#L43

Reply:
Yep agreed, that's what I meant, but typed it out wrong :)

Reply:
ah, gotcha. So we wouldn't be using the force enable then, right?

Reply:
Yeah, switched off of force enable.